addon/components/au-time-picker.js

```diff
@@ -120,4 +119,9 @@ export default class AuTimePickerComponent extends Component {
     this.secondValue = current.getSeconds();
     this.callBackParent(this.getTimeObject);
   }
+
+  formatTimeNumber(number) {
+    if (number < 10) return `0${number}`;
```

Comment:
You could also use [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) to skip the comparison.

Reply:
Nice, that would reduce this otherwise external helper to a one-liner.